FungusToast.postman_collection.json

```diff
@@ -0,0 +1,117 @@
+{
+	"info": {
+		"_postman_id": "0592e1e2-1d07-4c74-990a-08306d4cae06",
+		"name": "Fungus Toast",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Get User Games",
+			"protocolProfileBehavior": {
+				"disableBodyPruning": true
```

Comment:
I have no idea what this option does but it sounds like something my grandma would appreciate ![](https://media.giphy.com/media/2PDauUpCvUgxi/giphy.gif)